patches/chrome-installer-mac-signing-signing.py.patch

```diff
@@ -1,24 +1,31 @@
 diff --git a/chrome/installer/mac/signing/signing.py b/chrome/installer/mac/signing/signing.py
-index cd93959385debe65d04c0a4664f638173da54b75..898af6df3915f5770d0e7575eb747c4d14822f13 100644
+index cd93959385debe65d04c0a4664f638173da54b75..70a1d7741908ee9481bd2026ec1d493c086f7e7d 100644
 --- a/chrome/installer/mac/signing/signing.py
 +++ b/chrome/installer/mac/signing/signing.py
-@@ -11,6 +11,7 @@ import os.path
+@@ -8,9 +8,11 @@ bundle that need to be signed, as well as providing utilities to sign them.
+ 
+ import copy
+ import os.path
++import collections
  
  from . import commands
  from .model import CodeSignOptions, CodeSignedProduct, VerifyOptions
 +from signing_helper import AddBravePartsForSigning, GenerateBraveWidevineSigFile
  
  _PROVISIONPROFILE_EXT = '.provisionprofile'
  _PROVISIONPROFILE_DEST = 'embedded.provisionprofile'
-@@ -136,6 +137,7 @@ def get_parts(config):
+@@ -136,7 +138,9 @@ def get_parts(config):
              library_basename.replace('.dylib', ''),
              verify_options=VerifyOptions.DEEP)
  
-+    AddBravePartsForSigning(parts, config)
-     return parts
+-    return parts
++    ordered_parts = collections.OrderedDict(parts)
```

Comment:
these additional lines are not needed in this file and should be done internally in `AddBravePartsForSigning`